src/locales/hi.json

```diff
@@ -26,7 +26,8 @@
       "From": "से",
       "no-node": "कोई नोड चयनित नहीं",
       "index": "इंडेक्स",
-      "label": "लेबल"
+      "label": "लेबल",
```

Comment:
can you please make a root section follows component path "components/CodeEditor/MonacoEditor" and put your settings there as I cant see how these relate to Projects. Seems that Projects section is a dumping ground for things.